pkg/reconciliation/construct_service_test.go

```diff
@@ -74,3 +74,37 @@ func TestServiceNameGeneration(t *testing.T) {
 	service := newSeedServiceForCassandraDatacenter(dc)
 	assert.Equal(t, "notcool-bob-seed-service", service.Name)
 }
+
+func TestAddingAdditionalLabels(t *testing.T)  {
```

Comment:
It would be nice to see a test that demonstrates expected behavior when labels are added from both `AdditionalLabels` and `AdditionalServiceConfig`.

Reply:
Yeah I would add one. I was thinking that I did one. Forgive me.

Reply:
Done